tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -56,14 +56,14 @@ esac
 done
 eval set -- "$pps"
 
-
-if ! [ -x "$(command -v jq)" ]; then
-  >&2 echo "ABORTED: Required command is not on your PATH: jq."
-  >&2 echo "         Please install it before you continue."
-
-  exit 2
-fi
-
+for dependency in jq openssl
```

Comment:
i have always used `which` for this in the past - whats the difference?

Reply:
https://unix.stackexchange.com/a/85250 is a better explanation than we probably needed 🤓 In our case the two will behave the same as both are built in commands